projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -439,11 +439,11 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
     ];
 
     this.allCartesianData = [
-      ...this.allSeriesData,
       ...this.bands.map(
         band =>
           new CartesianBand(this.d3Utils, this.domRenderer, band, this.scaleBuilder, this.getTooltipTrackingStrategy())
-      )
+      ),
+      ...this.allSeriesData
```

Comment:
Seems unrelated? intentional?

Reply:
This is unrelated (sorry, I suck), but intentional. As I was working on this feature I was on a page that showed bands and I wanted to fix the front to back ordering since it was bothering me.